### DISCUSSION TITLE
免費升級N卡

### DISCUSSION THREAD

**Discussion by Ben6uo** [1101-GS4719-C/Python-Community-GS4719-C#584] [General]
N卡20系列以下都可以體驗"類光追"
https://www.nvidia.com/zh-tw/geforce/geforce-experience/download/
只要在程式中調整光線參數就可以體驗"類光追"

**Comment by jason79461385** [comment]
我記得好像有限定規格1650之類的，1050這種好像不行。

**Comment by alexthebigpp** [comment]
光追真的讚，之前原價買3070玩cyberpunk，畫面超美

**Comment by yenwei12138** [comment]
好讚!感謝分享，這樣玩遊戲整個舒服起來

**Comment by d26488399** [comment]
感謝分享，馬上試試看，希望我的顯卡再戰十年

**Comment by yucheng1028** [comment]
1060再戰五年
